logbook-spring/src/main/java/org/zalando/logbook/spring/RemoteResponse.java

```diff
@@ -105,7 +105,8 @@ private static final class Passing implements State {
     @Override
     public int getStatus() {
         try {
-            return response.getRawStatusCode();
+            // This may throw NPE
```

Comment:
Should we fallback to the same status on every implementation? I wouldn't say I like that in some modules it falls back to 200, but could a common practice. Or we could fallback to `-1` as a way to say "no data" provided.

I can't say I'm a fan of the idea that a logging library can fail requests because one of the attributes (even the one that's probably always there) is missing.

Reply:
We should definitely discuss it. I also thought about changing the interface to return `Integer` instead of `int`, but didn't consider its implications.

Reply:
FYI: Tried my approach (`Integer` instead of `int`) and failed: One of the classes implements a Spring interface which has `public int getStatus()`, and extends a class which would then have `public Integer getStatus()`, and bam! clash.

I don't like default `200` either (because it can be misleading), and would like to see some fallback like `-1` as you suggested.